<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>467</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TuPartido Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>941</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabAdminTickets">
     <attribute name="title">
      <string>Pagos</string>
     </attribute>
     <widget class="QTableView" name="tableViewTickets">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>921</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblSelectedDate">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>01-01-2018</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddDay">
      <property name="geometry">
       <rect>
        <x>178</x>
        <y>7</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSubDay">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>7</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPayTickets">
      <property name="geometry">
       <rect>
        <x>778</x>
        <y>7</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Marcar como pago</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAdminStadium">
     <attribute name="title">
      <string>Canchas</string>
     </attribute>
     <widget class="QLabel" name="lblComingSoonAdminStadium">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>391</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Proximamente vas a tener la opcion de gestionar tus canchas desde aca.</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBoxInformation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>941</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Informacion</string>
    </property>
    <widget class="QLabel" name="lblStadiumName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cargando...</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblLblStadiumName">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>30</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancha:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblDirection">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>30</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Cargando...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAyuda"/>
    <addaction name="actionAcerca"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="actionAcerca">
   <property name="text">
    <string>Acerca</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
